### ISSUE TITLE
Create a release

### ISSUE THREAD

**Issue by JosselinTD** [anrt-type/GoticoAntiqua#1]
Hi !

Awesome work on the fonts !

Can you make a "release" on github to ease the download for the non "github-aware" people interested on this work ?

Thanks !

**Comment by alexmyczko** [comment]
Yes please because versions like https://repology.org/project/fonts-gotico-antiqua/versions 0+gitYYYYMMDD are not really beautiful.

**Comment by kgfeuerherm** [comment]
That would be appreciated–I’m at a loss regarding how to download these.
